packages/react-native-reanimated/src/platformFunctions/dispatchCommand.ts

```diff
@@ -36,29 +36,39 @@ export let dispatchCommand: DispatchCommand;
 function dispatchCommandFabric(
   animatedRef: AnimatedRefOnJS | AnimatedRefOnUI,
   commandName: string,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   args: Array<unknown> = []
 ) {
-  'worklet';
+  /*  'worklet';
   if (!_WORKLET) {
     return;
   }
 
   const shadowNodeWrapper = animatedRef() as ShadowNodeWrapper;
   global._dispatchCommandFabric!(shadowNodeWrapper, commandName, args);
```

Comment:
We should also comment https://github.com/ExodusMovement/react-native-reanimated/blob/3c88e86afdee84aecd3e00828335dc306a3fb45e/packages/react-native-reanimated/Common/cpp/reanimated/RuntimeDecorators/UIRuntimeDecorator.cpp#L31-L32
and
https://github.com/ExodusMovement/react-native-reanimated/blob/3c88e86afdee84aecd3e00828335dc306a3fb45e/packages/react-native-reanimated/Common/cpp/reanimated/RuntimeDecorators/UIRuntimeDecorator.cpp#L37